e2e/composer-min-max-version/test.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+\PHPStan\Testing\assertType('int<80099, 80299>', PHP_VERSION_ID);
+\PHPStan\Testing\assertType('8', PHP_MAJOR_VERSION);
+\PHPStan\Testing\assertType('int<0, 2>', PHP_MINOR_VERSION);
```

Comment:
```suggestion
\PHPStan\Testing\assertType('int<80100, 80299>', PHP_VERSION_ID);
\PHPStan\Testing\assertType('8', PHP_MAJOR_VERSION);
\PHPStan\Testing\assertType('int<1, 2>', PHP_MINOR_VERSION);
```

AFAK `PHP_VERSION_ID` starts with the major/minor even if run on alpha release

`PHP_MINOR_VERSION` should be narrowed if `PHP_MAJOR_VERSION` is one constant number

Reply:
See https://github.com/phpstan/phpstan-src/pull/2968#discussion_r1524840873

Reply:
I saw it. But I do not support it. The reason is composer already check this during runtime and if the check is really, really wanted, the user should ignore that error instead.